pyogrio/geopandas.py

```diff
@@ -84,13 +86,39 @@ def read_dataframe(
         specific (e.g. typically 0 for Shapefile and 1 for GeoPackage, but can
         still depend on the specific file). The performance of reading a large
         number of features usings FIDs is also driver specific.
+    sql : str, optional (default: None)
+        The sql statement to execute. Look at the sql_dialect parameter for 
+        more information on the syntax to use for the query. When combined 
+        with other keywords like columns, skip_features, max_features, where 
+        or bbox, those are applied after the sql query. The ``layer`` keyword 
```

Comment:
Also `fids` keyword cannot be combined; add a note here and add to list of of unsupported parameters under `fids` parameter.